<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>240</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_forMainButton">
     <item>
      <widget class="QPushButton" name="pushButton_changeProfilePage">
       <property name="styleSheet">
        <string notr="true">background-color: #008080;
color: white;</string>
       </property>
       <property name="text">
        <string>Профіль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_openCaloriTableWindow">
       <property name="styleSheet">
        <string notr="true">background-color: #008080;
color: white;</string>
       </property>
       <property name="text">
        <string>Таблиця калорій</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_openCalorieCalcWindow">
       <property name="styleSheet">
        <string notr="true">background-color: #008080;
color: white;</string>
       </property>
       <property name="text">
        <string>Розрахувати калорійність страви</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_changeDishViewPage">
       <property name="styleSheet">
        <string notr="true">background-color: #008080;
color: white;</string>
       </property>
       <property name="text">
        <string>Перегляд власних страв</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_closeApp">
       <property name="styleSheet">
        <string notr="true">background-color: #008080;
color: white;</string>
       </property>
       <property name="text">
        <string>Вихід</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_Main">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>451</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="profile_page">
     <widget class="QPushButton" name="pushButton_changeCreateProfilePage">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>390</y>
        <width>161</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #008080;
color: white;</string>
      </property>
      <property name="text">
       <string>Створити профіль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_changeEditProfilePage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #008080;
color: white;</string>
      </property>
      <property name="text">
       <string>Редагувати профіль</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: white;
 border: 2px solid #CCCCCC; 
border-radius: 10px;
border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_Name">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #606060</string>
       </property>
       <property name="text">
        <string>Ім'я:</string>
       </property>
      </widget>
      <widget class="QLabel" name="profileNamelabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>0</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: 000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Weight">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #606060</string>
       </property>
       <property name="text">
        <string>Вага:</string>
       </property>
      </widget>
      <widget class="QLabel" name="profileWeightLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: 000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Plan">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #606060</string>
       </property>
       <property name="text">
        <string>План:</string>
       </property>
      </widget>
      <widget class="QLabel" name="profilePlanlabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: 000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Goal">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #606060</string>
       </property>
       <property name="text">
        <string>Ціль:</string>
       </property>
      </widget>
      <widget class="QLabel" name="profileGoalLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: 000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: white;
 border: 2px solid #CCCCCC; 
border-radius: 10px;
border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_calorieNorm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #606060</string>
       </property>
       <property name="text">
        <string>Дена норма калорій</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_caloricDayNorm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: 000000</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>210</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: white;
 border: 2px solid #CCCCCC; 
border-radius: 10px;
border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_proteinNorm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #606060</string>
       </property>
       <property name="text">
        <string>Дена норма білків</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_proteinDayNorm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: 000000</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: white;
 border: 2px solid #CCCCCC; 
border-radius: 10px;
border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_carbonohydrateNorm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #606060</string>
       </property>
       <property name="text">
        <string>Дена норма вуглеводів</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_carbohydrateDayNorm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: 000000</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: white;
 border: 2px solid #CCCCCC; 
border-radius: 10px;
border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_fatNorm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #606060</string>
       </property>
       <property name="text">
        <string>Дена норма жирів</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_fatDayNorm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: 000000;
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="profileEdit_page">
     <widget class="QLabel" name="label_editGoal">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Введіть бажаний результат(кг)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_editUserName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_editName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Введіть ім'я</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_editWeight">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Введіть вагу(кг)</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_editUserPlan">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_editUserWeight">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_editUserGoal">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_editPlan">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Оберіть план</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_editProfile">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>380</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #008080;
color: white;</string>
      </property>
      <property name="text">
       <string>Редагувати профіль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cancelProfileEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #008080;
color: white;</string>
      </property>
      <property name="text">
       <string>Скасувати</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="profileCreate_page">
     <widget class="QLineEdit" name="lineEdit_enterUserName">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_enterUserWeight">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_enterUserGoal">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_enterWeight">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Введіть вашу вагу(кг)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_choosePlan">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Оберіть план</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_enterName">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Введіть ваше ім'я</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_enretGoal">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Введіть бажаний результат(кг)</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_changeUserPlan">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_createProfile">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>380</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #008080;
color: white;</string>
      </property>
      <property name="text">
       <string>Створити профіль</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
